python/feathub/feature_tables/tests/test_mysql_source_sink.py

```diff
@@ -0,0 +1,180 @@
+        self.client = self.get_client(
+            extra_config={"processor": {"flink": {"native.parallelism.default": "1"}}}
+        )
+
+        input_data = pd.DataFrame(
+            [
+                [1, 1, datetime(2022, 1, 1, 0, 0, 0).strftime("%Y-%m-%d %H:%M:%S")],
+                [2, 2, datetime(2022, 1, 1, 0, 0, 1).strftime("%Y-%m-%d %H:%M:%S")],
+                [3, 3, datetime(2022, 1, 1, 0, 0, 2).strftime("%Y-%m-%d %H:%M:%S")],
+                [1, 4, datetime(2022, 1, 1, 0, 0, 3).strftime("%Y-%m-%d %H:%M:%S")],
+            ],
+            columns=["id", "val", "ts"],
+        )
+
+        schema = (
+            Schema.new_builder()
+            .column("id", types.Int64)
+            .column("val", types.Int64)
+            .column("ts", types.String)
+            .build()
+        )
+
+        source = self.create_file_source(
+            input_data,
+            keys=["id"],
+            schema=schema,
+            timestamp_field="ts",
+            timestamp_format="%Y-%m-%d %H:%M:%S",
+        )
+
+        table_name = self._testMethodName
+
+        engine = sqlalchemy.create_engine(self.mysql_container.get_connection_url())
+        with engine.begin() as conn:
+            conn.execute(
+                sqlalchemy.text(
+                    f"CREATE TABLE {table_name} ("
+                    f"id BIGINT PRIMARY KEY, val BIGINT, ts TEXT)"
+                )
+            )
+
+        sink = self._get_mysql_sink(table_name)
+
+        self.client.materialize_features(
+            features=source,
+            sink=sink,
+            allow_overwrite=True,
+        ).wait(30000)
+
+        expected_result = (
+            pd.DataFrame(
+                [
+                    [2, 2, datetime(2022, 1, 1, 0, 0, 1).strftime("%Y-%m-%d %H:%M:%S")],
+                    [3, 3, datetime(2022, 1, 1, 0, 0, 2).strftime("%Y-%m-%d %H:%M:%S")],
+                    [1, 4, datetime(2022, 1, 1, 0, 0, 3).strftime("%Y-%m-%d %H:%M:%S")],
+                ],
+                columns=["id", "val", "ts"],
+            )
+            .sort_values(by=["id"])
+            .reset_index(drop=True)
+        )
+
+        with engine.begin() as conn:
+            result = conn.execute(sqlalchemy.text(f"SELECT * FROM {table_name}"))
+            r = result.fetchall()
+        result_df = (
+            pd.DataFrame(data=r, columns=["id", "val", "ts"])
+            .sort_values(by=["id"])
+            .reset_index(drop=True)
+        )
+        print(result_df)
+        assert_frame_equal(expected_result, result_df)
+
+    def _get_mysql_sink(self, table_name):
+        return MySQLSink(
+            host=self.mysql_container.get_container_host_ip(),
+            port=int(
+                self.mysql_container.get_exposed_port(
+                    self.mysql_container.port_to_expose
+                )
+            ),
+            database=self.mysql_container.MYSQL_DATABASE,
+            username=self.mysql_container.MYSQL_USER,
```

Comment:
Can we make the order of parameters consistent with their order in the function declaration?